maxim/logger/llamaindex/__init__.py

```
@@ -0,0 +1,114 @@
+from llama_index.core.callbacks import BaseCallbackHandler, CBEventType, EventPayload
+from llama_index.core.llms import ChatMessage
+
+from ..logger import GenerationConfig, GenerationError, Logger, TraceConfig
+from ..models import TraceContainer
+from ...scribe import scribe
+
+
+class MaximLlamaIndexCallbackHandler(BaseCallbackHandler):
+    """Callback handler that logs LlamaIndex LLM calls to Maxim."""
+
+    def __init__(self, logger: Logger):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)